packages/grid/x-data-grid-pro/src/tests/columns.DataGridPro.test.tsx

```diff
@@ -420,4 +421,18 @@ describe('<DataGridPro /> - Columns', () => {
       ]);
     });
   });
+
+  it('should throw if unsupported column type is used', function test() {
```

Comment:
We don't skip, we can keep it anonymous

```suggestion
  it('should throw if unsupported column type is used', () => {
```